src/test/java/org/sonar/plugins/clojure/sensors/ancient/AncientOutputParserTest.java

```diff
@@ -0,0 +1,36 @@
+package org.sonar.plugins.clojure.sensors.ancient;
+
+import org.junit.Test;
+import org.sonar.plugins.clojure.sensors.CommandStreamConsumer;
+
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+
+public class AncientOutputParserTest {
```

Comment:
It would be nice if we could have a test to check when there is not match for the regex during the parse.